pkg/operation/common/utils.go

```diff
@@ -561,47 +561,6 @@ func DeleteAlertmanager(ctx context.Context, k8sClient client.Client, namespace
 	return kubernetesutils.DeleteObjects(ctx, k8sClient, objs...)
 }
 
-// DeletePlutono deletes the monitoring stack for the shoot owner.
-func DeletePlutono(ctx context.Context, k8sClient kubernetes.Interface, namespace string) error {
```

Comment:
Just to double check because I haven't tested the migration from the helm chart to the golang component in a local setup during my review:
do all the artifacts that were previously deployed by the helm chart have the same name as the components deployed by the managed resource, so that no old artefacts are leaked after the migration?